app/internal/page/enter_reference_number.go

```diff
@@ -1,28 +1,28 @@
-package attorney
+package page
 
 import (
 	"encoding/base64"
 	"errors"
 	"net/http"
 
 	"github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
+	"github.com/gorilla/sessions"
 	"github.com/ministryofjustice/opg-go-common/template"
 	"github.com/ministryofjustice/opg-modernising-lpa/app/internal/actor"
 	"github.com/ministryofjustice/opg-modernising-lpa/app/internal/dynamo"
-	"github.com/ministryofjustice/opg-modernising-lpa/app/internal/page"
 	"github.com/ministryofjustice/opg-modernising-lpa/app/internal/sesh"
 	"github.com/ministryofjustice/opg-modernising-lpa/app/internal/validation"
 )
 
 type enterReferenceNumberData struct {
-	App    page.AppData
+	App    AppData
 	Errors validation.List
 	Form   *enterReferenceNumberForm
-	Lpa    *page.Lpa
+	Lpa    *Lpa
 }
 
-func EnterReferenceNumber(tmpl template.Template, shareCodeStore ShareCodeStore, sessionStore SessionStore, attorneyStore AttorneyStore) page.Handler {
-	return func(appData page.AppData, w http.ResponseWriter, r *http.Request) error {
+func EnterReferenceNumber(tmpl template.Template, shareCodeStore ShareCodeStore, sessionStore sessions.Store, certificateProviderStore CertificateProviderStore, attorneyStore AttorneyStore, actorType actor.Type) Handler {
```

Comment:
it's kind of weird passing both stores in, I wouldn't mind this handler being duplicated for now as it is a bit custom for each

Reply:
I guess you're right - felt nice to be able to fully drop all the custom CP handler.